CHANGELOG.rst

```diff
@@ -1,8 +1,8 @@
 XCommon CMake Change Log
 ========================
 
-UNRELEASED
-----------
+1.2.0
+-----
 
   * ADDED:     Support for multiple prebuilt archives in a static library repository
   * ADDED:     Support for configuring compiler flags on static library additional sources
```

Comment:
might be worth adding/clarifying "matching current xcommon_cmake version" or similar